components/badge.css

```diff
@@ -0,0 +1,68 @@
+.badge-showcase {
+	display: flex;
+	justify-content: baseline;
+	align-items: flex-end;
+	border: 2px solid var(--teritiary-color);
+	border-radius: 8px;
+	padding: 3rem 0;
+}
+
+.badge--image {
+	width: 9.4rem;
+	height: 9.4rem;
+	border-radius: 50%;
+	margin: 2.4rem;
+	position: relative;
+}
+.badge-status {
+	height: 2.2rem;
+	width: 2.2rem;
+	border-radius: 50%;
+	position: absolute;
+	margin-top: -57px;
+	margin-left: 100px;
+	border: 2px solid white;
+}
+
+.badge--status-active {
+	background-color: #00ff00;
+}
+
+.badge--status-offline {
+	background-color: #ff0000;
+}
+
+.badge--status-away {
+	background-color: #313531ea;
+}
+
+/*******badge icon css****/
+
+.badge-icons-container {
+	display: flex;
+	justify-content: center;
```

Comment:
Define, display flex and its properties as an utility class and use it across everywhere